src/webots/gui/WbStreamingServer.cpp

```diff
@@ -206,7 +206,7 @@ void WbStreamingServer::onNewTcpData() {
   const QString &line(socket->peek(1024));  // Peek the request header to determine the requested url.
   QStringList tokens = QString(line).split(QRegExp("[ \r\n][ \r\n]*"));
   if (tokens[0] == "GET") {
-    const QString &requestedUrl(tokens[1].replace(QRegExp("^/"), ""));
+    QString requestedUrl(tokens[1].replace(QRegExp("^/"), ""));
```

Comment:
Why did you removed the const qualifier here?

Reply:
That's a mistake. I just restored it.